server/routes/api.js

```diff
@@ -11,7 +11,7 @@ import stats from "./stats";
 const router = Router();
 
 router.get("/", (_, res) => {
-	pool.query("SELECT * FROM person", (err, result) => {
+	pool.query("SELECT * FROM person WHERE NOT blocked", (err, result) => {
```

Comment:
```suggestion
	pool.query("SELECT * FROM person", (err, result) => {
```
Can we leave this as is? what if they block someone by mistake and need to unblock?